### DISCUSSION TITLE
need help/explanation about implementing GET _dbs_info startkey (json)

### DISCUSSION THREAD

**Discussion by bugsysBugs** [apache/couchdb#5300] [Q&A] [answered]
I'm trying to use startkey param in "GET _dbs_info" but no matter what I try I always get all the DBs. I don't understand how the startkey in query param can be represented in json.( according to docs )
My databases are named as: 
domain/somainName/subDomainName/subSubdomainName
I'm streaming and agregating ( in node.js ) and need to be able to make requests on user action like: startkey="domain/", starkey="domain/subDomain", startkey="domain/subDomain/subSubDomain" etc.
Thanks in advance for your time.

**Comment by big-r81** [comment, marked as answer]
Hi,

how did you query your DB?

I can't reproduce this. For example, i have two dbs.

Request:
```
curl -s -u a:a "http://127.0.0.1:15984/_all_dbs"
```
Response:
```json
[
  "_replicator",
  "_users"
]
```
If I want only the '_users' db in the result, I would do the following.

Request:
```
curl -s -u a:a 'http://127.0.0.1:15984/_dbs_info?startkey="_users"'
```
Response:
```json
[
  {
    "key": "_users",
    "info": {
      "instance_start_time": "1728841527",
      "db_name": "_users",
      "purge_seq": "0-g1AAAABTeJzLYWBgYMpgTmHgz8tPSTV0MDQy1zMAQsMckEQeC5BkaABS_4EgK5EBp8pEhqR6iJIsACz0Fj4",
      "update_seq": "1-g1AAAACTeJzLYWBgYMpgTmHgz8tPSTV0MDQy1zMAQsMckEQeC5BkaABS_4EgK4M5kSEXKMCeapZmnmxoiakLp0mJDEn1UCMYwUaYGJqZp5qkYSrPAgArkCRR",
      "sizes": {
        "file": 12652,
        "external": 5387,
        "active": 2342
      },
      "props": {},
      "doc_del_count": 0,
      "doc_count": 1,
      "disk_format_version": 8,
      "compact_running": false,
      "cluster": {
        "q": 2,
        "n": 1,
        "w": 1,
        "r": 1
      }
    }
  }
]
```

**Reply by bugsysBugs** [reply]
Yes, it's working with curl like a charm. I'm using axios, using an node.js proxy and also requesting a stream ... Must be something there. Thanks a lot for your time - now at least I know where to debug.

**Reply by bugsysBugs** [reply]
Was the proxy - didn't handle correctly the query params - thanks again!
